plugin/sampling/strategystore/adaptive/aggregator.go

```diff
@@ -112,9 +129,23 @@ func (a *aggregator) RecordThroughput(service, operation string, samplerType spa
 
 func (a *aggregator) Start() {
 	go a.runAggregationLoop()
+	a.postAggregator.Start()
```

Comment:
Both of these run on a timer. need to check if on the same interval. If the same we can combine into one bg goroutine, just call post-aggregator after each tick.

Reply:
Yes, they both have the same interval. I have made some changes; please have a look.
& 
we still need `a.postAggregator.Start()` to start election participant